olm/update-nightly-olm-files.sh

```diff
@@ -33,13 +33,26 @@ do
     echo "   - Updating new package version with roles defined in: ${role}"
     cp "$role" generated/current-role.yaml
     operator-sdk olm-catalog gen-csv --csv-version "${newNightlyPackageVersion}" --from-version="${lastPackageVersion}" 2>&1 | sed -e 's/^/      /'
+
+    containerImage=$(sed -n 's|^ *image: *\([^ ]*/che-operator:[^ ]*\) *|\1|p' "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml")
+    createdAt=$(date -u +%FT%TZ)
+    echo "   - Updating new package version fields:"
+    echo "       - containerImage => ${containerImage}" 
+    echo "       - createdAt => ${createdAt}" 
+    sed \
+    -e "s|containerImage:.*$|containerImage: ${containerImage}|" \
+    -e "s/createdAt:.*$/createdAt: \"${createdAt}\"/" \
+    "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml" \
+    > "${packageFolderPath}/${newNightlyPackageVersion}/${packageName}.v${newNightlyPackageVersion}.clusterserviceversion.yaml.new"
+    
```

Comment:
if you use `-i` you can apply the sed transform in-place to the file, without having to redirect to .new then move .new to the actual filename.

`sed -i -e "s..." filename`

Reply:
Well, it seemed to me that the `-i` option doesn't work the same between MacOS and linux. But maybe I'm wrong.